test/core/iomgr/threadpool_test.cc

```diff
@@ -0,0 +1,173 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "src/core/lib/iomgr/executor/threadpool.h"
+
+#include "test/core/util/test_config.h"
+
+#define SMALL_THREAD_POOL_SIZE 20
```

Comment:
As per https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros, let us try to avoid defining macros. Use const variables instead.